src/main/kotlin/net/fabricmc/bot/extensions/mappings/MappingsManager.kt

```diff
@@ -91,7 +93,41 @@ class MappingsManager {
     suspend fun getMethodMappings(minecraftVersion: String, query: String): List<MappingsResult>? {
         val mappings = openMappings(minecraftVersion) ?: return null
 
-        return getMappingsResults(mappings, query, ClassDef::getMethods)
+        return getMappingsResults(mappings, preProcessMethodQuery(query), ClassDef::getMethods)
+    }
+
+    /**
+     * Preprocesses a method mapping query, adapting the query value to make more ergonomic lookups possible.
+     */
+    private fun preProcessMethodQuery(query: String): String {
+        /*
```

Comment:
This should be in the doc comment for the function — split it from the summary with two newlines